config/storybook-preset-carbon/index.js

```diff
@@ -26,15 +26,14 @@ module.exports = {
     '@storybook/addon-storysource',
     '@storybook/addon-knobs',
     '@storybook/addon-actions',
-    '@storybook/addon-links',
     '@storybook/addon-docs',
     '@storybook/addon-notes/register',
     'storybook-readme/register',
-
-    // Phase 3: port over custom panels/add-ons
+    '@storybook/addon-links',
+    require.resolve('./dist/preset.js'),
```

Comment:
Should this only be included conditionally based on the Custom Properties ENV variable?